addons/web_editor/static/lib/odoo-editor/test/spec/editor.test.js

```diff
@@ -2622,9 +2622,31 @@ X[]
                     contentAfter: `<p>[]<br></p>`,
                 });
                 await testEditor(BasicEditor, {
-                    contentBefore: `<h1>[]abcd</h1>`,
+                    contentBefore: `<h1><br>[]</h1>`,
                     stepFunction: deleteBackward,
-                    contentAfter: `<p>[]abcd</p>`,
+                    contentAfter: `<p>[]<br></p>`,
```

Comment:
So we have a change of spec then because it seems this is what we wanted originally?
cc @dmo-odoo @sebgeelen @pierre-pde

Reply:
@Zynton , IMO this change is correct. 
It's better this new behavior :)